lego/utils/management/commands/bump_last_login.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from django.utils import timezone
+
+from lego.apps.users.models import User
+from lego.utils.management_command import BaseCommand
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
Is this maybe better suited as a data migration instead? Then everyone gets it, including when running locally.

Is there a use case of this after we run it, or?

Reply:
I have been thinking and I am really struggling to find a use case for running this multiple times. A data migration is the way to go.